src/gui/org/deidentifier/arx/gui/resources/messages.properties

```diff
@@ -936,6 +936,10 @@ DialogOrderSelection.12=Import...
 DialogOrderSelection.13=Export... 
 DialogOrderSelection.14=OK 
 DialogOrderSelection.15=Cancel 
+DialogOrderSelection.16=Import failed
+DialogOrderSelection.17=The file does not contain the values of the attribute's domain. Please select a file that contains only the values of the attribute's domain.
+DialogOrderSelection.18=Import failed
+DialogOrderSelection.19=The file contains more values than present in the attribute's domain. Please select a file that contains only the values of the attribute's domain.
```

Comment:
(1) This would be possible, but would result in an inconsistency as DialogOrderSelection.10, 3, 5 and 8 are also titles from error messages. Therefore, I would opt for creating a new seperate title message.
(2) Done